test/unit/next-swc.test.ts

```diff
@@ -107,7 +107,7 @@ describe('next/swc', () => {
             if (n === "Map" || n === "Set") return Array.from(n);
             if (n === "Arguments" || /^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(n)) return _array_like_to_array(o, minLen);
         }
-        import { useState } from "react";
+        import { useState } from 'react';
```

Comment:
Where is this single quote stuff coming from?
Everything else in the output has double quotes (as one would expect)

Reply:
The windows tests are failing because of this as well

Reply:
The new swc version used the original style as written in source code

Reply:
From the single quote in the input.

https://github.com/vercel/next.js/blob/578c400a0435de5ff40922feb4a08d84a4d76b31/test/unit/next-swc.test.ts#L79